app/templates/create.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
+  
+  <form class="container my-1 rounded bg-white border border-primary" id="createForm" data-link="{{ url_for('create') }}">
+    {{ form.hidden_tag() }}
+
+    <div class="row my-3 gy-2">
+      {{ form.title.label(class="col-form-label col-2") }}
+      <div class="col-10">
+        {{ form.title(class="form-control") }}
+      </div>
+      {% for error in form.title.errors %}
+      <span style="color: red;">[{{ error }}]</span>
+      {% endfor %}
+    </div>
+
+    <div class="row my-3 gy-2">
+      {{ form.start.label(class="col-form-label col-2") }}
+      <div class="col-10">
+        {{form.start(class="form-control") }}
+      </div>
+      {% for error in form.start.errors %}
+      <span style="color: red;">[{{ error }}]</span>
+      {% endfor %}
+    </div>
+
+    <div class="row my-3 gy-2">
+      {{ form.destination.label(class="col-form-label col-2") }}
+      <div class="col-10">
+        {{ form.destination(class="form-control") }}
+      </div>
+      {% for error in form.destination.errors %}
+      <span style="color: red;">[{{ error }}]</span>
+      {% endfor %}
+    </div>
+
+    <div class="row my-3 gy-2">
+      {{ form.time.label(class="col-form-label col-2") }}
+      <div class="col-10">
+        {{ form.time(class="form-control") }}
+      </div>
+      {% for error in form.time.errors %}
+      <span style="color: red;">[{{ error }}]</span>
+      {% endfor %}
+    </div>
+
+    <div class="row my-3 gy-2">
+      {{ form.attempts.label(class="col-form-label col-2") }}
+      <div class="col-10">
+        {{ form.attempts(class="form-control") }}
+      </div>
+      {% for error in form.attempts.errors %}
+      <span style="color: red;">[{{ error }}]</span>
+      {% endfor %}
+    </div>
+
+    <div class="my-3">{{ form.submit(class="btn btn-primary") }}</div>
+  </form>
```

Comment:
Part of me feels like for such a big repeat of mildly edited code there *should* be a jinja way to put it in afor loop using the form class. Not necessary for this issue, but potentially one that could be looked into

Reply:
I looked into it and there is. You can simply loop through the fields of the form.  I have now made this change to the challenge creation template.